code/modules/mob/living/living.dm

```diff
@@ -286,9 +286,18 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
-							"<span class='warning'>[src] grabs you [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by your hands":"passively"]!</span>", null, null, src)
-			to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>")
+			if(zone_selected == BODY_ZONE_PRECISE_GROIN && istype(getorganslot(ORGAN_SLOT_TAIL), /obj/item/organ/tail)) // austation begin -- tail entwining, this time with catgirl/human ships in mind too
```

Comment:
`zone_selected == BODY_ZONE_PRECISE_GROIN`

noerp

Reply:
tail goes in the groin, doris 😳